### ISSUE TITLE
Fire aspect and Knockback not working

### ISSUE THREAD

**Issue by hasanas83V2001** [Speentie8081/SmashHit#3]
Both fire aspect and knockback enchants on a sword lose functionality, they just don't work at all, please help, the only plugins I'm running are ProtocolLib and SmashHit, I'm using spigot 1.7.10 b1649 with the latest patch.

**Comment by Speentie8081** [comment]
I do not believe these are implemented to work yet, also I am not the original developer.
I will look into this problem though, if I ever come back to work on this project again.
